README.rst

```diff
@@ -179,6 +186,62 @@ $ curl https://httpbin.org/get?show\_env=1
       "url": "http://httpbin.org/get?show_env=1"
     }
 
+$ curl -XPOST https://httpbin.org/file/dir1/dir2?name=myfile.json
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+::
+
+    {
+      "file_path": "/tmp/dir1/dir2/myfile.json"
```

Comment:
Note: The root directory for saving files is a configuration value.  It will default to `tmp`